src/components/Profile/AuthorizedProfile.tsx

```diff
@@ -1,112 +1,209 @@
 import { signOut } from "firebase/auth";
 import { useState } from "react";
-import { StyleSheet, View, Text, Button, Switch } from "react-native";
-import Icon from "react-native-vector-icons/FontAwesome";
+import {
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
+  Modal,
+  Alert,
+  Pressable,
+} from "react-native";
+import { default as FAIcon } from "react-native-vector-icons/FontAwesome";
 import { firebaseAuth } from "../../../firebaseConfig";
+import getLocationFromDevice from "@/utils/getLocationFromDevice";
+import { LinearGradient } from "expo-linear-gradient";
+import { default as FeatherIcon } from "react-native-vector-icons/Feather";
+import useUser from "@/hooks/useUser";
 
-export default function AuthorizedProfile() {
-  const [pushIsEnabled, setPushIsEnabled] = useState(false);
-  const pushNotificationsSwitch = () => {
-    setPushIsEnabled((previousState) => !previousState);
-    if (!pushIsEnabled) {
-      console.log("Push notifications are ON");
-    } else {
-      console.log("Push notifications are OFF");
-    }
-  };
-
-  const [emailIsEnabled, setEmailIsEnabled] = useState(false);
-  const emailNotificationsSwitch = () => {
-    setEmailIsEnabled((previousState) => !previousState);
-    if (!emailIsEnabled) {
-      console.log("Email notifications are ON");
-    } else {
-      console.log("Email notifications are OFF");
-    }
-  };
+const gradientColors = ["#FFD18A", "#C6B9E4", "#81A8F4"];
+const colorsLocations = [0.15, 0.65, 1];
 
+export default function AuthorizedProfile() {
+  const [modalVisible, setModalVisible] = useState(false);
```

Comment:
```suggestion
export default function AuthorizedProfile() {
  const [isModalVisible, setIsModalVisible] = useState(false);
```
Boolean variable names should start with is